[AG-50383] Added a new article on ad blocking issues in Opera (#822)
AdguardTeam/KnowledgeBase: docs/adguard-browser-extension/solving-problems/ads-not-blocked-in-opera.md

---
title: Ad blocking issues on search websites in Opera
sidebar_position: 3
---

> If AdGuard Browser Extension for Opera fails to block ads on search websites, either automatically or manually, this indicates that the extension does not have permission to access search page results.

Reviewer: ```suggestion

If AdGuard Browser Extension fails to block ads on search websites in Opera, either automatically or manually, it means that the extension does not have permission to access search page results.
```
